Marlin/src/module/stepper.cpp

```diff
@@ -2763,15 +2787,18 @@ hal_timer_t Stepper::block_phase_isr() {
 
       // Initialize the trapezoid generator from the current block.
       #if ENABLED(LIN_ADVANCE)
-        la_active = (current_block->la_advance_rate != 0);
         #if DISABLED(MIXING_EXTRUDER) && E_STEPPERS > 1
           // If the now active extruder wasn't in use during the last move, its pressure is most likely gone.
           if (stepper_extruder != last_moved_extruder) la_advance_steps = 0;
```

Comment:
`la_advance_steps` does not exist for `SMOOTH_LIN_ADV`.

Reply:
Wow, good catch. Fixed by putting the whole block under 
`#if ENABLED(LIN_ADVANCE) && DISABLED(SMOOTH_LIN_ADV)`

Reply:
For some reason github is not showing the block as outdated, but now line 2789 is 
`#if ENABLED(LIN_ADVANCE) && DISABLED(SMOOTH_LIN_ADV)`